papergrid/src/lib.rs

```diff
@@ -664,23 +668,31 @@ impl Default for Style {
         Self {
             alignment_h: AlignmentHorizontal::Left,
             alignment_v: AlignmentVertical::Top,
-            indent: Indent {
+            padding: Padding {
                 bottom: 0,
+                bottom_char: ' ',
                 left: 0,
+                left_char: ' ',
                 right: 0,
+                right_char: ' ',
                 top: 0,
+                top_char: ' ',
             },
             span: 1,
         }
     }
 }
 
 #[derive(Debug, Clone)]
-pub struct Indent {
+pub struct Padding {
     pub top: usize,
+    pub top_char: char,
     pub bottom: usize,
+    pub bottom_char: char,
     pub left: usize,
+    pub left_char: char,
     pub right: usize,
+    pub right_char: char,
 }
```

Comment:
How about making it 

```rust
pub struct Padding {
    pub top: Indent,
    pub bottom: Indent,
    pub left: Indent,
    pub right: Indent,
}

pub struct Indent {
    pub symbol: char,
    pub size: usize,
}
```

Reply:
👋🏻 separation looks better 
But I'm not sure about `Indent` naming 👉🏻 does "Indent" should contains symbol? Maybe choose fitter name?

Reply:
Indeed I may be bad in naming :)

But what is a name you suggest?

`Space`
`Area`

I don't know I honestly think Indent is OK as it's more general word.
indent - a [space](https://dictionary.cambridge.org/ru/%D1%81%D0%BB%D0%BE%D0%B2%D0%B0%D1%80%D1%8C/%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9/space) at the [edge](https://dictionary.cambridge.org/ru/%D1%81%D0%BB%D0%BE%D0%B2%D0%B0%D1%80%D1%8C/%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9/edge) or on the [surface](https://dictionary.cambridge.org/ru/%D1%81%D0%BB%D0%BE%D0%B2%D0%B0%D1%80%D1%8C/%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9/surface) of something.

Maybe rename `symbol` to `fill`.